nncf/experimental/post_training/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -0,0 +1,305 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from os import stat
+from typing import Dict, Tuple
+from typing import List
+from typing import TypeVar
+from typing import Union
+
+import numpy as np
+from nncf.common.graph import NNCFGraph
+from nncf.common.graph import NNCFNode
+from nncf.common.tensor import NNCFTensor
+from nncf.common.utils.logger import logger as nncf_logger
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.graph.transformations.layout import TransformationLayout
+from nncf.common.utils.backend import BackendType, get_backend
+from nncf.common.graph.transformations.commands import TargetPoint
+from nncf.common.tensor_statistics.collectors import TensorStatisticCollectorBase
+from nncf.experimental.post_training.algorithms import AlgorithmParameters
+from nncf.experimental.post_training.algorithms.algorithm import Algorithm
+from nncf.experimental.post_training.algorithms.algorithm import PostTrainingAlgorithms
+from nncf.experimental.post_training.algorithms.fast_bias_correction.onnx_backend import ONNXFBCAlgoBackend
+from nncf.experimental.post_training.algorithms.fast_bias_correction.backend import ALGO_BACKENDS
+from nncf.experimental.post_training.api.engine import Engine
+from nncf.experimental.post_training.factories import NNCFGraphFactory
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPoint
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPointsContainer
+
+ModelType = TypeVar('ModelType')
+
+# pylint: disable=protected-access
+
+
+class FastBiasCorrectionParameters(AlgorithmParameters):
+    """
+    Base class of FastBiasCorrection parameters.
+    """
+
+    def __init__(self, number_samples: int = 100, threshold: float = 2.0):
+        self.number_samples = number_samples
+        self.threshold = threshold
+
+    def to_json(self) -> Dict[str, Union[str, float, int]]:
+        """
+        Serialize all FastBiasCorrection parameters to JSON.
+        """
```

Comment:
Could you please add the `:return:` section? Why body of this method is empty?

Reply:
Originally this method was added to be able to dump the parameters of algorithms which were used during network compression. This should be useful for debugging purpose, esp. for accuracy aware pipelines

Reply:
@kshpv, but at the same time, there is no detailed example of the implementation or at least output JSON format. 
How could we use this method without any implementation? Why do we need this method in the base `AlgorithmParameters` class?